common/src/main/java/com/selfdot/cobblemonmegas/common/command/MigrateMegaStone.java

```diff
@@ -0,0 +1,42 @@
+package com.selfdot.cobblemonmegas.common.command;
+
+import com.mojang.brigadier.Command;
+import com.mojang.brigadier.context.CommandContext;
+import com.mojang.brigadier.exceptions.CommandSyntaxException;
+import com.selfdot.cobblemonmegas.common.item.MegaStoneHeldItemManager;
+import net.minecraft.item.ItemStack;
+import net.minecraft.server.command.ServerCommandSource;
+import net.minecraft.server.network.ServerPlayerEntity;
+import net.minecraft.text.Text;
+import net.minecraft.util.Hand;
+
+public class MigrateMegaStone implements Command<ServerCommandSource> {
```

Comment:
Added this due to necessity on PokeClash to migrate player's megastones from previous mod, up to your discretion if you want to include this

Reply:
Absolutely, which mod does it migrate from?

Reply:
Nevermind, just realised you said AaronMusk's.